extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -1552,4 +1556,122 @@ describes.sandboxed('amp-ad-network-doubleclick-impl', {}, () => {
       });
     });
   });
+
+  describe('#correlator clear experiment', () => {
+    let onVisibilityChangedHandler;
+    let visabilityState;
+
+    beforeEach(() => {
+      onVisibilityChangedHandler = null;
+      visabilityState = VisibilityState.PAUSED;
+      return createIframePromise().then(fixture => {
+        const viewer = {
+          isVisible: () => true,
+          onVisibilityChanged: handler => {
+            onVisibilityChangedHandler = handler;
+          },
+          getVisibilityState: () => visabilityState,
+          whenFirstVisible: () => Promise.resolve(),
+        };
+        sandbox.stub(Services, 'viewerForDoc').returns(viewer);
+        element = createElementWithAttributes(fixture.doc, 'amp-ad', {
+          type: 'doubleclick',
+          height: '250',
+          width: '320',
+        });
+        document.body.appendChild(element);
+        impl = new AmpAdNetworkDoubleclickImpl(element);
+        impl.win.ampAdPageCorrelator = 12345;
+        delete impl.win['dbclk_a4a_viz_change'];
+      });
+    });
+
+    it('clears if in experiment', () => {
+      impl.win.ampAdPageCorrelator = 12345;
```

Comment:
can delete this line, it's in the beforeEach.  I'd just replace with an expect for what it should be.